src/main.c

```diff
@@ -48,6 +48,8 @@ char __noinit __stack cdcacm_setup_stack_area[CDCACM_STACKSIZE];
 char __noinit __stack baudrate_reset_stack_area[RESET_STACKSIZE];
 char __noinit __stack usb_serial_stack_area[USBSERIAL_STACKSIZE];
 
+const char *firmware_version="CODK-M_1.6.0";
```

Comment:
We might need to talk about the format of this version number, shouldn't we be using something like a combination of the BLE version & the actual name of the firmware version, so it matches Git tags? i.e.

BLE-1.5.7_Bootes

(I just made up the BLE version, dunno what it really is)